package.json

```diff
@@ -60,6 +60,7 @@
 		"unset-value": "^2.0.1"
 	},
 	"devDependencies": {
+    "@aws-sdk/client-cloudwatch": "^3.441.0",
```

Comment:
The formatting of this file is slightly off, and [we recommend using pinned dependencies](https://github.com/guardian/recommendations/blob/main/dependencies.md#javascript), so we’ll want to drop that leading caret (`^`).

Reply:
Fixed in https://github.com/guardian/frontend/pull/26675/commits/0197055144e94e66d1fcb92cc8422ef9d085f773